emcc.py

```diff
@@ -476,7 +477,6 @@ def default_min_browser_version(browser, version):
 
   if settings.WASM_BIGINT:
     default_min_browser_version('Safari', 150000)
-    default_min_browser_version('Edge', 79)
```

Comment:
unrelated change?

Reply:
Actually its needed, since setting the edge version to anything by -1 currently means no features are supported.  i.e. the edge version currently means only EdgeHTML and not modern Edge.

Reply:
The proposal is that for now the edge version only refers to the old edge browser, and if the features of modern edge ever diverge from chromium we can start to support `MIN_EDGE_VERSION` being distinct from `MIN_CHROME_VERSION`

Not mentioning edge at all there just means that default of "-1" (not supported at all) will prevail.

Right now I'm not considering edge at all in the feature_matix.py because we currently assume that folks can use the MIN_CHROMIUM_VERSION to target modern versions of edge.    Instead of removing this line I could instead extend feature_matrix.py to include support for MIN_EDGE_VERSION (but it would just be identical what is already there for chromium).

Reply:
OK, I reworked `feature_matrix.py` such that this change is no longer needed.

Reply:
Oh, then the Chrome version refers to modern Edge too? Sgtm, but I wasn't aware their features had even been in sync so far. Is that the case in practice?

Reply:
If you looks at the versions in https://en.wikipedia.org/wiki/Microsoft_Edge (expand "New Edge release history") you will see that edge versions correspond 1-to-1 with blink versions.. to from a JS/wasm perspective one would expect them to always match.